src/services/storage.js

```diff
@@ -1,9 +1,33 @@
 import {createState} from '@hookstate/core'
 
-const starCount = createState(5);
+const starCount = createState(0);
+const currentIndex = createState(null);
+const favouriteMovies = createState([]);
 
-const increaseStarCount = () => {
-    starCount.set(p => p + 1);
+const addFavourite = (movie) => {
+    favouriteMovies.set([...favouriteMovies, movie]);
+    console.log(favouriteMovies.get());
+};
```

Comment:
The first: the issue with adding unique movie indexes to favouriteMovies.  
The second: problem with passing created state to useState and use to return the array of indexes.  (useState must be used within the body of a function component). 
The third: add unique indexes to the array (I thought to use "new  Set() while adding the movie to avoid duplicating of indexes")